docs/concepts/flatcar-container-linux.md

```diff
@@ -61,3 +61,37 @@ Kubelet mount points on Flatcar Container Linux:
 | /etc/resolv.conf           | /etc/resolv.conf           |           |
 | /opt/cni/bin               | /opt/cni/bin               |           |
 
+### Customization
+
+Flatcar Container Linux can be customized via [Container Linux Configs](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/configuration.md) (CLC)
```

Comment:
I'd use https://docs.flatcar-linux.org/container-linux-config-transpiler/doc/examples/ instead of linking to the old CL website.